sklego/preprocessing.py

```diff
@@ -53,6 +54,48 @@ def _check_X_for_type(X):
             raise TypeError("Provided variable X is not of type pandas.DataFrame")
 
 
+class DateTimeFeatures(TransformerMixin, BaseEstimator):
+    """
+    Add common time-series features to data frame in your scikit-learn pipeline.
+    All time features supported by pandas.dt can be added (e.g. weekday, weekofyear, year,
+    quarter, month, day, hour and minute. By default it will generate all the time features. They
+    can be individually excluded.
+
+    :param date_col: name of the date column (default= 'date')
+    :param date_features: tuple of dt features
+
+    """
+
+    def __init__(self, date_col="date", date_features=("weekday", "weekofyear", "year", "quarter",
```

Comment:
kind of being picky here but ... "hour" ... is that a `date_feature`?

Reply:
not entirely, but the function is called 'DateTimeFeature' ... I could rename this attribute to datetime_features as well :)